### PR TITLE
added name explanation onto README

### PR DESCRIPTION
**Pull request by danbugs**
~been meaning to add this for a bit, and finally got around to it.

Signed-off-by: Dan Chiarlone <dchiarlone@microsoft.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +58,12 @@ WASI's primary goal is to provide operating system abstractions for Wasm applica
 ### What is the difference between SpiderLightning and Dapr?
 
 SpiderLightning and Dapr share the same goal of providing capabilities to distributed applications, but each project's approach to achieving this goal is very different. For example, while Dapr runs as a sidecar container and provides an HTTP/gRPC interface for applications, SpiderLightning's chooses not take a dependency on network transport protocols. Instead, `slight` links applications directly to generated WASM bindings, then executes them directly.
+
+### Why the name "SpiderLightning"?
+
+<figure style="float:right;margin:.1em">
+    <img src="docs/images/readme2.jpg" width="320px" />
```
Comment (Mossaka) [inline README.md:65]: How about we make the picture BIGGER

**Review by Mossaka — APPROVED** [review]
lgtm!

**Review by devigned — APPROVED** [review]
lgtm! However, I would totally support a larger picture.

**Comment by danbugs** [comment]
I've fixed the image display, merging now!

**Pull request merged**